stacks/aws/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ resources:
   - ../../admission-webhook/webhook/v3
   - ../../common/centraldashboard/overlays/stacks
   - ../../kubeflow-roles/base
-  - ../../jupyter/jupyter-web-app/base_v3
+  - ../../jupyter/jupyter-web-app/overlays/aws
```

Comment:
@PatrickXYS Can you have a separate application with aws overlay under /stack/aws without modifying the jupyter overlay?

Reply:
That's the overlay we created before for ECR image. 

I'll make change in `/stack/aws`, in the future, we'll remove this overlay under `jupyter-web-app` component.

Reply:
Fixed.